openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -60,15 +63,15 @@ <h4 class="question__title ui-toggle-visibility__control">
                                         </h4>
 
                                         {% if criterion.options %}
-                                            <ul class="question__answers ui-toggle-visibility__content has--hints">
+                                            <ul class="question__answers ui-slidable__content has--hints" aria-labelledby="oa_grade_{{ xblock_id }}_criteria_{{ criterion_num }}" id="oa_grade_{{ xblock_id }}_criteria_{{ criterion_num }}_content">
                                                 {% for assessment in criterion.assessments %}
                                                     {% include "openassessmentblock/grade/oa_assessment_title.html" with assessment=assessment %}
                                                 {% endfor %}
                                             </ul>
                                         {% endif %}
 
                                         {% if criterion.has_feedback %}
-                                            <ul class="question__answers ui-toggle-visibility__content has--hints">
+                                            <ul class="question__answers ui-slidable__content has--hints">
```

Comment:
I wonder if this needs any sort of "aria linking". We can wait and ask during the accessibility review.